app/src/main/kotlin/org/eurofurence/connavigator/ui/FragmentViewDealer.kt

```diff
@@ -76,12 +74,18 @@ class FragmentViewDealer() : Fragment(), ContentAPI, HasDb, AnkoLogger {
                         "This artist did not supply any artist description to show to you :("
             )
 
-            ui.aboutArt.loadMarkdown(
-                    if (dealer.aboutTheArtText.isNotEmpty())
-                        dealer.aboutTheArtText
-                    else
-                        "this artist did not supply any art descriptions to show to you :("
-            )
+            if (dealer.artPreviewImageId == null) {
```

Comment:
Yeah, but the photoView is nested in a layout with padding (that also contains the art preview text).
`ui.artPreview.visibility = View.GONE` may be obsolete here, but you still want the fragment to hide that verticalLayout container (with padding) if both image is missing and text is empty; otherwise it results in a weird green "bar" to the user, which is an empty container with padding.